Package.swift

```diff
@@ -18,10 +18,10 @@ let package = Package(
   ],
   targets: [
     .plugin(
-      name: "AirbnbSwiftFormatPlugin",
+      name: "FormatSwift",
       capability: .command(
         intent: .custom(
-          verb: "format",
+          verb: "airbnb-format-swift",
```

Comment:
When another package consumes this plugin (e.g. https://github.com/airbnb/lottie-ios/blob/97c05f8272fc8e3a2cefb3d41c6bcb90c6d28739/Rakefile#L108) it uses this name. Let's pick a more specific name to avoid collisions. I'm not sure what happens when the names collide.

Reply:
I think `swift package format` is noticeably more ergonomic that `swift package airbnb-format-swift`. I'm not sure how worried we need to be about plugin command conflicts -- perhaps we can hold off on trying to work around that unless it comes up in the future?

Reply:
I tend to agree with Cal here, I really dislike using `-` or `_` on the CLI since it's much more inconvenient to type.

Reply:
I'm following the [example](https://developer.apple.com/videos/play/wwdc2022/110401/) of the WWDC talk on making these plugins.

Reply:
> I'm not sure how worried we need to be about plugin command conflicts -- perhaps we can hold off on trying to work around that unless it comes up in the future?

Reply:
Interesting, it looks like we may not need to worry about conflicts ([source](https://github.com/apple/swift-evolution/blob/main/proposals/0332-swiftpm-command-plugins.md)).
<img width="1127" alt="Screen Shot 2022-07-21 at 1 37 07 PM" src="https://user-images.githubusercontent.com/1791049/180310713-f083520e-acf5-47dd-b098-29d4ccae8d0b.png">

Reply:
aha cool! So I guess it would be `swift package AirbnbSwift:format` in that case.

Reply:
definitely way better than `swift package AirbnbSwiftFormatPlugin:format`, that was a good suggestion